src/app/templates/templates-routes.module.ts

```diff
@@ -0,0 +1,17 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+
+import { TemplatesModule } from './templates.module';
+import { TemplatesListPage } from './list-page/list-page.component';
+
+const routes: Routes = [
+  { path: '', component: TemplatesListPage, pathMatch: 'full' },
+];
+
+@NgModule({
+  imports: [
+    RouterModule.forChild(routes),
+    TemplatesModule,
+  ],
+})
+export class TemplateRoutesModule { }
```

Comment:
I will say that having 'templates' in the file name helps me out a little only because my most common tool to switch between files doesn't default to showing the path :-)

And +1 camel case filename.  And I suspect we'll hit this with integrations too.